app/_src/gateway/production/debug-request.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Debug requests
```

Comment:
❗ For reviewers: I did not test the examples because of the short turn around time. I can make a ticket for post-release to test these unless you think testing these is a blocker for docs release.

Reply:
I think we should definitely test this guide, but we can publish for now and revisit - as long as we don't forget to do it.

Reply:
Okay, made a Jira ticket for the week of the 13th to test this